app/components/after_sign_off_component.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @challenged %>
+  <p>
+   Application is now in assessment and assigned to <%= @accessor_name %>
+  </p>
+<% else %>
+  <%= form_with model: @planning_application,
+    url: determine_planning_application_path(@planning_application),
+    local: true,
+    builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+    <div class="govuk-button-group">
+      <%= link_to "Publish decision",
+                  publish_planning_application_path(@planning_application),
+                  aria: { describedby: "publish-completed" },
+                  class: "govuk-button" %>
```

Comment:
They want the button to link to the determination page where they do the final publishing. :-|

Confusing isn't it? A button that is a link. That cost me a lot of time because I couldn't get in contact with the design team. So I did indeed try to make it a form - you have to add a publishing date to get it to work. But when I contacted them it turned out they just wanted a link.

Reply:
oh I see what you mean... I should remove the form completely... I think